thorlcr/thorutil/thormisc.cpp

```diff
@@ -74,22 +74,25 @@ static Owned<IMPtagAllocator> ClusterMPAllocator;
 
 // stat. mappings shared between master and slave activities
 const StatisticsMapping spillStatistics({StTimeSpillElapsed, StTimeSortElapsed, StNumSpills, StSizeSpillFile});
+const StatisticsMapping jhtreeCacheStatistics({ StNumNodeCacheAdds, StNumLeafCacheAdds, StNumBlobCacheAdds, StNumNodeCacheHits, StNumLeafCacheHits, StNumBlobCacheHits, StCycleNodeLoadCycles, StCycleLeafLoadCycles,
```

Comment:
I think either these stats should be added to indexReadStatistics, or them merged with this.

The 2 existing places that use indexReadStatistics look wrong (passed to queryThor().queryFileCache().lookupIFileIO) - which looks like it should not override the default stats mapping (diskLocalStatistics) used by CLazyFileIO.